.github/workflows/electron-passkeys.yml

```
@@ -0,0 +1,113 @@
+      - 'packages/electron-passkeys/**'
+      - '.github/workflows/electron-passkeys.yml'
+
+permissions:
+  contents: read
+  actions: read
+
+concurrency:
+  group: electron-passkeys-${{ github.head_ref || github.ref }}
+  cancel-in-progress: true
+
+permissions:
```

actionlint (1.7.12)

[error] key "permissions" is duplicated in "workflow" section. previously defined at line:11,col:1

(syntax-check)